integration_testing/features/delete-multiple-items-permanently.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Delete multiple items permanently
+
+	Background: 
+		Given That I have removed items from my channel editor tree
+
+	Scenario: Delete multiple items permanently
+		When I click the *Trash* button in the top appbar
+		Then I see a list of items removed from my channel
+		When I select multiple items from the list
+			And I click *Delete* in the bottom bar
+		Then I see a 'WARNING' message appear

```

Comment:
We do not capitalize full words or phrases in the scenarios, even if though they appear styled so in the UI.